pkg/modules/notary/node.go

```diff
@@ -0,0 +1,58 @@
+package notary
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/encoding/address"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neo-go/pkg/wallet"
+	"go.uber.org/zap"
+)
+
+// UpdateNotaryNodes implements Notary interface and updates current notary account.
+func (n *Notary) UpdateNotaryNodes(notaryNodes keys.PublicKeys) {
+	n.accMtx.Lock()
+	defer n.accMtx.Unlock()
+
+	if n.currAccount != nil {
+		var isPresent bool
+		for _, node := range notaryNodes {
+			if node.Equal(n.currAccount.PrivateKey().PublicKey()) {
+				isPresent = true
```

Comment:
Well, if you're returning `if isPresent` you might as well do it here.

Reply:
Fixed.